### DISCUSSION TITLE
Unable to deploy to production environment

### DISCUSSION THREAD

**Discussion by Cvr421** [vercel/next.js#44220] [Help]
I am using nextjs app
Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error 
while deploying to production . i have tried lot of things but still i am gettig the same error
Please anyone have a solution please share it

**Comment by icyJoseph** [comment]
Well there should be more logs or reasons printed to the console during build time. 

Where are you deploying your app?

A good check list would be to:

- make sure env vars are available
- test if it fails to build locally as well (next build && next start)
- Are there any 3rd party servers that cannot be reached during build time?
